src/obs_inv_utils/discover_interface.py

```diff
@@ -0,0 +1,293 @@
+
+# The purpose of this script is to 1) allow obs_inv_utils to inventory files from NASA Discover
+
+
+import os
+import subprocess
+import re
+from pathlib import Path
+from collections import namedtuple, OrderedDict
+import attr
+from datetime import datetime
+
+nl = '\n'
+
+# DiscoverCommand adapted from hpss_io_interface.HpssCommand
+DiscoverCommand = namedtuple(
+    'DiscoverCommand',
+    [
+        'command',
+        'arg_validator',
+        'output_parser'
+    ],
+)
+
+# DiscoverCommandRawResponse adapted from hpss_io_interface.HppsCommandRawResponse
+DiscoverCommandRawResponse = namedtuple(
+    'DiscoverCommandRawResponse',
+    [
+        'command',
+        'return_code',
+        'error',
+        'output',
+        'success',
+        'args_0',
+        'submitted_at',
+        'latency'
+    ],
+)
+
+# DiscoverListContents adapted from aws_s3_interface.AwsS3ObjectsListContents
+DiscoverListContents = namedtuple(
+    'DiscoverListContents',
+    [
+        'prefix',
+        'files_count',
+        'files_meta',
+        'obs_cycle_time',
+        'submitted_at',
+        'latency'
+    ],
+)
+
+HpssTarballContents = namedtuple(
```

Comment:
is this actually getting used in this file or was it just there for reference making the discover ones? if just reference they can be removed now